src/components/modals/StakingModal.tsx

```diff
@@ -99,8 +65,9 @@ const StakingModal = ({
   const allowDelegatedStaking =
     gateway?.settings.allowDelegatedStaking ?? false;
 
-  const delegateData = walletAddress
-    ? gateway?.delegates[walletAddress?.toString()]
+  const delegateData: AoGatewayDelegate | undefined = walletAddress
+    ? // @ts-expect-error - delegates is currently available on the gateway
+      gateway?.delegates[walletAddress?.toString()]
```

Comment:
this can be replaced with `getGatewayDelegates` on the ar-io-sdk in v2.4.0-alpha.5+

Reply:
Punting on this, will use new handlers to retrieve stakes/withdrawals by owner when available.